### PR TITLE
fix(wat.tv): remove `MYTF1` context to remove "Permission insuffisante" error

### LINKED ISSUE #7303: [TF1] [wat.tv]: Permission insuffisante
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

France

### Provide a description that is worded well enough to be understood

when i launch the command : yt-dlp  https://www.tf1.fr/tmc/une-nounou-d-enfer/videos/une-nounou-denfer-s03-e7-un-homme-metamorphose-01883134.html
I have a message: 
[TF1] une-nounou-denfer-s03-e7-un-homme-metamorphose-01883134: Downloading JSON metadata
[wat.tv] Extracting URL: wat:13992774
[wat.tv] 13992774: Downloading JSON metadata
ERROR: [wat.tv] 13992774: Permission insuffisante

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.tf1.fr/tmc/une-nounou-d-enfer/videos/une-nounou-denfer-s03-e7-un-homme-metamorphose-01883134.html']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d] (zip)
[debug] Python 3.10.6 (CPython x86_64 64bit) - Linux-5.15.0-73-generic-x86_64-with-glibc2.35 (OpenSSL 3.0.2 15 Mar 2022, glibc 2.35)
[debug] exe versions: ffmpeg 4.4.2 (setts), ffprobe 4.4.2, phantomjs broken
[debug] Optional libraries: Cryptodome-3.17, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, secretstorage-3.3.1, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1788 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
Current Build Hash: a818cd85814f94bb7a07b6a731a5f110093e0549590625558985c244c0a31e0e
yt-dlp is up to date (stable@2023.03.04)
[TF1] Extracting URL: https://www.tf1.fr/tmc/une-nounou-d-enfer/videos/une-nounou-denfer-s03-e7-un-homme-metamorphose-01883134.html
[TF1] une-nounou-denfer-s03-e7-un-homme-metamorphose-01883134: Downloading JSON metadata
[wat.tv] Extracting URL: wat:13992774
[wat.tv] 13992774: Downloading JSON metadata
ERROR: [wat.tv] 13992774: Permission insuffisante
  File "/home/ludovic/.local/bin/yt-dlp/yt_dlp/extractor/common.py", line 694, in extract
    ie_result = self._real_extract(url)
  File "/home/ludovic/.local/bin/yt-dlp/yt_dlp/extractor/wat.py", line 64, in _real_extract
    raise ExtractorError(error_desc, expected=True)
```

### PR DESCRIPTION
**Pull request by davinkevin**
### Description of your *pull request* and other information

Modification in the `wat.py` extractor to be compatible with changes applied by the platform. See issue #7303 for more detail. I've just applied modification tested in this issue and locally. 

Fixes #7303

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details><details><summary>Copilot Summary</summary>  

### Summary
🛠️🔄❌

Fix wat.tv extraction by updating URL parameters in `yt_dlp/extractor/wat.py`

> _`wat.tv` changed API_
> _`pver` updated, `context` dropped_
> _video extraction fixed_

### Walkthrough
* Remove `context` and update `pver` in JSON URL to fix video extraction from wat.tv ([link](https://github.com/yt-dlp/yt-dlp/pull/7898/files?diff=unified&w=0#diff-45cafb67cb871bcf2d966a0fa5b23b1062a8381d0fb25cc37683c72151c011bcL57-R57))

</details>

### CODE REVIEW

**Comment by bashonly** [comment]
Add a test (non-DRM)

Also, do you see any way we can catch the `"Permission insuffisante"` error for DRM-protected videos and `report_drm()` instead?

**Comment by davinkevin** [comment]
> Add a test, unless the first test is still non-DRM and working (and if so, check if it needs to be updated)

All links are very old (~10 years old) and not working anymore… for a long time. 
I can "try" to add test, but I'm out of my expertise area, so it will be way more complicated than this 8 keys modification. 

Do you have a simple command to run test just for this extractor?

**Comment by bashonly** [comment]
> Do you have a simple command to run test just for this extractor?

to run the 1st extractor test:
```
python3 test/test_download.py TestDownload.test_Wat
```
2nd and 3rd and so forth:
```
python3 test/test_download.py TestDownload.test_Wat_1
python3 test/test_download.py TestDownload.test_Wat_2
```
or run them all
```
python3 test/test_download.py TestDownload.test_Wat_all
```

> I can "try" to add test, but I'm out of my expertise area, so it will be way more complicated than this 8 keys modification.

Add a dict to the `_TESTS` list, start simply like:
```py
    {
        'url': 'https://www.tf1.fr/tmc/quotidien-avec-yann-barthes/videos/selah-sue-when-it-all-falls-down-en-live-pour-quotidien-67219751.html',
        'info_dict': {
            'id': '67219751',
            'ext': 'mp4',
        },
    }
```
Then run the test. It will fail and output the fields that are missing or incorrect. Copy those into your test dict, and run again. Repeat until the test passes.

**Comment by davinkevin** [comment]
New test added 👍

**Review by bashonly — APPROVED** [review]
Had to do a little fixup on the tests; I had given some bad instruction about adding a `TF1` test to the `Wat` extractor. In the future, please don't force push. You can push as many commits as you want to your PRs. Thanks for adding the test!

**Comment by davinkevin** [comment]
I've modified the test to be on `wat` when I've seen the format and behaviour was not correct.
And sorry, I forced push to keep only 1 commit (I really don't like "fix…" commit in the history), and because it's my own branch, I just `--amend` it over.

**Comment by bashonly** [comment]
> And sorry, I forced push to keep only 1 commit (I really don't like "fix…" commit in the history), and because it's my own branch, I just --amend it over.

What I'm saying is: don't do this. It makes it difficult to review the PR.

**Comment by davinkevin** [comment]
Ok, noted for future PR here 👍

File: yt_dlp/extractor/wat.py [hunk 1]
```diff
@@ -41,6 +41,18 @@ class WatIE(InfoExtractor):
...
+            'url': 'wat:14010600',
+            'info_dict': {
+                'id': '14010600',
+                'title': 'Burger Quiz - S03 EP21 avec Eye Haidara, Anne Depétrini, Jonathan Zaccaï et Pio Marmaï',
+                'thumbnail': 'https://photos.tf1.fr/1280/720/burger-quiz-11-9adb79-0@1x.jpg',
+                'upload_date': '20230819',
+                'duration': 2312,
+                'ext': 'mp4',
+            },
+            'params': {'skip_download': 'all'},
```
Comment (bashonly) [inline yt_dlp/extractor/wat.py:54]:
```suggestion
            'params': {'skip_download': 'm3u8'},
```

**Comment by davinkevin** [comment]
Thanks and sorry for the commit conflicts! 😇

**Pull request merged**